File: seccion_6/reto.java
class Facultad {
    void fundar() {
        System.out.println("La facultad de Informatica fue fundada en el año 2000");
    }
}

class Universidad {
    private Facultad facultad;

    Universidad() {
        this.facultad = new Facultad();
    }

    void fundar() {
        facultad.fundar();
        System.out.println("La universidad fue fundada en el año 1990");
    }
}

public class reto {
    public static void main(String[] args) {
        Universidad u = new Universidad();
        u.fundar();
    }
}

File: seccion_6/reto.py
class facultad:
    def fundar(self):
        print("La facultad de Informatica fue fundada en el año 2000")

class universidad:
    def __init__(self):
        self.facultad = facultad()

    def fundar(self):
        self.facultad.fundar()
        print("La universidad fue fundada en el año 1990")

u = universidad()
u.fundar()